plugins/check_apt_upgrade

```diff
@@ -94,8 +94,9 @@ EOF
 hwe_support_status () {
   [ -e "$PATH_TO_HWE_SUPPORT_STATUS" ] || return
 
-  HWE_SUPPORT_STATUS_MSG="; $($PATH_TO_HWE_SUPPORT_STATUS --$verbosity 2>&1)"
-  if [ "$?" != 0 ]; then
+  #HWE_SUPPORT_STATUS_MSG="; $($PATH_TO_HWE_SUPPORT_STATUS --$verbosity 2>&1)"
+  #if [ "$?" != 0 ]; then
```

Comment:
Would you mind dropping the commented out version, please?

Reply:
Done, see  345efd4 :)